### PR TITLE
fixup! feat(nutanix): add nutanix node popin

### PR DESCRIPTION
**Pull request by tibs245**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nutanix-scale-in-out`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-15577 #MANAGER-15557
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

File: packages/manager/modules/nutanix/src/dashboard/general-info/template.html [hunk 1]
```diff
@@ -183,7 +183,11 @@
                     </div>
                 </oui-tile-description>
                 <oui-action-menu data-compact data-placement="end">
-                    <oui-action-menu-item on-click="$ctrl.goToAddNode()">
+                    <oui-action-menu-item
+                        on-click="$ctrl.goToAddNode()"
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/template.html:187]:
```suggestion
                        data-on-click="$ctrl.goToAddNode()"
```

File: packages/manager/modules/nutanix/src/dashboard/nodes/list/controller.js [hunk 2]
```diff
@@ -1,18 +1,27 @@
 import { NODE_BADGE_STATE } from './constants';
+import { MAX_NODES_BY_CLUSTER } from '../../../constants';
 
 export default class NutanixAllNodesCtrl {
   /* @ngInject */
   constructor(ovhManagerRegionService, $translate) {
     this.ovhManagerRegionService = ovhManagerRegionService;
     this.$translate = $translate;
     this.NODE_BADGE_STATE = NODE_BADGE_STATE;
+    this.MAX_NODES_BY_CLUSTER = MAX_NODES_BY_CLUSTER;
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/nodes/list/controller.js:10]:
```suggestion
```

File: packages/manager/modules/nutanix/src/dashboard/nodes/list/controller.js [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
   $onInit() {
     const uniqueStates = [...new Set(this.nodes.map(({ state }) => state))];
     this.mapNodes = this.mapAllNodes();
 
+    this.addNodeTooltipContent =
+      this.nodes.length >= this.MAX_NODES_BY_CLUSTER
+        ? this.$translate.instant(
+            'nutanix_dashboard_nodes_add_node_max_node_tooltip',
+          )
+        : null;
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/nodes/list/controller.js:23]:
```suggestion
   this.isMaxNodesReached = this.nodes.length >= MAX_NODES_BY_CLUSTER
    this.addNodeTooltipContent =
       this.isMaxNodesReached
        ? this.$translate.instant(
            'nutanix_dashboard_nodes_add_node_max_node_tooltip',
          )
        : null;
```

File: packages/manager/modules/nutanix/src/dashboard/general-info/template.html [hunk 4]
```diff
@@ -183,7 +183,11 @@
...
+                        data-disabled="$ctrl.nodesDetails.length >= $ctrl.MAX_NODES_BY_CLUSTER"
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/template.html:188]: In this way, you compute the deactivation only one time and in one place
```suggestion
                        data-disabled="$ctrl.isMaxNodesReached"
```

File: packages/manager/modules/nutanix/src/dashboard/general-info/controller.js [hunk 5]
```diff
@@ -28,7 +32,9 @@ export default class NutanixGeneralInfoCtrl {
     this.REPLICATION_FACTOR_PREFIX = REPLICATION_FACTOR_PREFIX;
     this.NUTANIX_PERSONAL_LICENSE_EDITION = NUTANIX_PERSONAL_LICENSE_EDITION;
     this.GENERAL_INFO_TILE_TITLE = GENERAL_INFO_TILE_TITLE;
+    this.MAX_NODES_BY_CLUSTER = MAX_NODES_BY_CLUSTER;
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/controller.js:35]:
```suggestion
```

File: packages/manager/modules/nutanix/src/dashboard/general-info/controller.js [hunk 6]
```diff
@@ -68,6 +74,12 @@ export default class NutanixGeneralInfoCtrl {
     return this.NutanixService.getNodesWithState(this.serviceName)
       .then((nodesDetails) => {
         this.nodesDetails = nodesDetails;
+        this.addNodeTooltipContent =
+          nodesDetails.length >= this.MAX_NODES_BY_CLUSTER
+            ? this.$translate.instant(
+                'nutanix_dashboard_cluster_add_node_max_node_tooltip',
+              )
+            : null;
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/general-info/controller.js:82]:
```suggestion
        this.isMaxNodesReached = nodesDetails.length >= MAX_NODES_BY_CLUSTER;
        this.addNodeTooltipContent =
          this.isMaxNodesReached
            ? this.$translate.instant(
                'nutanix_dashboard_cluster_add_node_max_node_tooltip',
              )
            : null;
```

File: packages/manager/modules/nutanix/src/dashboard/nodes/list/template.html [hunk 7]
```diff
@@ -1,6 +1,9 @@
 <oui-datagrid rows="$ctrl.nodesMapped" data-page-size="25" class="mt-5">
     <oui-datagrid-topbar>
-        <oui-button on-click="$ctrl.goToAddNode()"
+        <oui-button
+            on-click="$ctrl.goToAddNode()"
+            data-disabled="$ctrl.nodes.length >= $ctrl.MAX_NODES_BY_CLUSTER"
```
Comment (darsene) [inline packages/manager/modules/nutanix/src/dashboard/nodes/list/template.html:5]:
```suggestion
            data-on-click="$ctrl.goToAddNode()"
            data-disabled="$ctrl.isMaxNodesReached"
```

**Review by ThibaudCrespin — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**